### ISSUE TITLE
Really strange results on a single teddy bear image.

### ISSUE THREAD

**Issue by Sharpiless** [liuyuan-pal/SyncDreamer#35]
![teddy](https://github.com/liuyuan-pal/SyncDreamer/assets/50802787/8dc72b76-ed4a-4ae0-b494-77b3b0727be3)

run:
```
python generate.py --ckpt ckpt/syncdreamer-pretrain.ckpt \
                   --input testset/teddy.png \
                   --output output/teddy2 \
                   --sample_num 4 \
                   --cfg_scale 0.0 \
                   --elevation 30 \
                   --crop_size 200
```

results:

![0](https://github.com/liuyuan-pal/SyncDreamer/assets/50802787/2b1ab0e9-1ba9-4e9a-b9f6-0822cabf0e71)
![1](https://github.com/liuyuan-pal/SyncDreamer/assets/50802787/9930a7f3-0050-4fd8-8218-2d4247458456)
![2](https://github.com/liuyuan-pal/SyncDreamer/assets/50802787/d64b2fc7-5172-43bc-8d5e-429c6fcde1d0)
![3](https://github.com/liuyuan-pal/SyncDreamer/assets/50802787/903d57fc-16e9-46b6-bd7d-216bb1fce52c)

run:
```
python generate.py --ckpt ckpt/syncdreamer-pretrain.ckpt \
                   --input testset/teddy.png \
                   --output output/teddy2 \
                   --sample_num 4 \
                   --cfg_scale 0.0 \
                   --elevation 0 \
                   --crop_size 200
```

results:
![0](https://github.com/liuyuan-pal/SyncDreamer/assets/50802787/681fe8ab-5272-4374-b43e-bff9681c2c77)
![1](https://github.com/liuyuan-pal/SyncDreamer/assets/50802787/87f3e81c-d8da-4fb4-80d3-292a9b837850)
![2](https://github.com/liuyuan-pal/SyncDreamer/assets/50802787/9a505d92-f8cf-4259-95dd-d82d1dddb72c)
![3](https://github.com/liuyuan-pal/SyncDreamer/assets/50802787/7430d044-e5c4-4c71-871d-bd9008c98c78)

**Comment by Sharpiless** [comment]
Only work well on the previded demo:
![0](https://github.com/liuyuan-pal/SyncDreamer/assets/50802787/99da37bc-724f-4682-b5ba-68be6591aa48)

Any suggestions would be appreciated.

**Comment by liuyuan-pal** [comment]
Hi, you may check if the segmentation of the input image is correct or not.

**Comment by Sharpiless** [comment]
> Hi, you may check if the segmentation of the input image is correct or not.

This is the segmented rgb-a image. The mask seems to be correct.
![](https://user-images.githubusercontent.com/50802787/273371891-8dc72b76-ed4a-4ae0-b494-77b3b0727be3.png)

**Comment by liuyuan-pal** [comment]
Hi, I've tried the image on the online demo https://huggingface.co/spaces/liuyuan-pal/SyncDreamer
![image](https://github.com/liuyuan-pal/SyncDreamer/assets/23974453/3c71dbc2-3717-4907-b90a-32ecb8a4b914)
It seems to be OK.

**Comment by liuyuan-pal** [comment]
oh, you need to use cfg scale at 1.5 or 2.0. and the image should have an elevstion of 0 degree.

**Comment by Sharpiless** [comment]
Thanks a lot!

**Comment by Sharpiless** [comment]
Another trivia question: Are you willing to share the code that generates the "[camera-16.pkl](https://github.com/liuyuan-pal/SyncDreamer/blob/main/meta_info/camera-16.pkl)" file? Thanks!

**Comment by Sharpiless** [comment]
> Another trivia question: Are you willing to share the code that generates the "[camera-16.pkl](https://github.com/liuyuan-pal/SyncDreamer/blob/main/meta_info/camera-16.pkl)" file? Thanks!

fixed by using blender 2.90 and https://github.com/liuyuan-pal/SyncDreamer/blob/main/blender_script.py

**Comment by liuyuan-pal** [comment]
Hi, it's this line https://github.com/liuyuan-pal/SyncDreamer/blob/1fa5ecf118cbda2096978d27dd602d4d027e065d/blender_script.py#L279
